pkgs/top-level/perl-packages.nix

```diff
@@ -10948,13 +10948,14 @@ let
     };
   };
 
-  LaTeXML = buildPerlPackage rec {
+  LaTeXML = lib.recursiveUpdate (buildPerlPackage rec {
     pname = "LaTeXML";
     version = "0.8.5";
     src = fetchurl {
       url = "mirror://cpan/authors/id/B/BR/BRMILLER/${pname}-${version}.tar.gz";
       sha256 = "0dr69rgl4si9i9ww1r4dc7apgb7y6f7ih808w4g0924cvz823s0x";
     };
+    outputs = [ "out" "tex" ];
```

Comment:
Is there an actual benefit to using multiple outputs here?

Reply:
Will answer this myself: It's to split bin from run, I guess.

Reply:
Indeed, it's the same pattern used in `auctex` and `noweb`.